types/dom-to-image-more.d.ts

```
@@ -0,0 +1,4 @@
+declare module "dom-to-image-more" {
+  import domToImage = require("dom-to-image-more");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)